src/models/MediaImage.ts

```diff
@@ -46,6 +46,48 @@ export class MediaImageL {
     get app(): Maybe<CushyAppL> {return this.draft?.app} // prettier-ignore
     get script(): Maybe<CushyScriptL> {return this.app?.script } // prettier-ignore
 
+    copyToClipboard = async () => {
+        const canvas = document.createElement('canvas')
+        const ctx = canvas.getContext('2d')
+
+        const img = new Image()
+        img.src = URL.createObjectURL(this.getAsBlob())
+
+        img.onload = () => {
+            canvas.width = img.width
+            canvas.height = img.height
+            ctx?.drawImage(img, 0, 0)
+
+            const dataUrl = canvas.toDataURL('image/png')
```

Comment:
good attempt going though the dataURL, then mapping over every char, but there is probably a better way to go using

`canvas.toBlob()`

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob#syntax

```ts
canvas.toBlob(callback)
canvas.toBlob(callback, type)
canvas.toBlob(callback, type, quality)
```